Minimal/app_config.yaml

```diff
@@ -1,9 +1,28 @@
-env_name: edge-minimal-example
-edm_deps:
-- click
-- pip
+image_name: edge-minimal-example
+framework: generic
+app_id: null
+app_version: 1.2.0
+python_version: 3.8
+repository: quay.io/enthought
```

Comment:
technically speaking this is the repository but part of the URI to the repo. Should be keep the repository server as quay.io and have the image_name being `enthought/edge-minimal-example` ? That is more in line with how images are typically described out there.

Reply:
Technically we have three parts:  the server "quay.io", the repository "enthought", and the image name "edge-minimal-example" -- with the "enthought" and "edge-minimal-example" combined for a full repository name (in our current quay.io use).  It might make sense to specify it in three parts instead of two.

Reply:
We can surely do three parts. The enthought part is the organisation(both on dockerhub and quay). The image name and the repository are the same.